jcli/src/jcli_lib/rest/v1/vote/mod.rs

```diff
@@ -0,0 +1,58 @@
+use crate::jcli_lib::rest::{Error, RestArgs};
+use crate::jcli_lib::utils::OutputFormat;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct AccountVotes {
+    #[structopt(flatten)]
+    args: RestArgs,
+
+    #[structopt(flatten)]
+    output_format: OutputFormat,
+
+    /// Account address to filter votes
+    #[structopt(short, long)]
+    account: String,
```

Comment:
This has been a positional parameter in other commands, e.g. `v0 account get`. IDK if this should be a consistent pattern, though.

Reply:
I'm not sure, in this case we have two domain-specific arguments and I don't think there's any predefined order. In that sense named parameters gives a bit more flexibility to the user.  In any case, I don't have a strong opinion, I guess either one is fine